controllers/newContent.controller.js

```diff
@@ -0,0 +1,131 @@
+const Content = require("../Models/contentSchema");
```

Comment:
The file name should just be named `content.controller.js` and not `newContent.controller.js`